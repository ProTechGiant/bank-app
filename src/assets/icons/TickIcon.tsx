import { Path, Svg } from "react-native-svg";

import { IconProps } from "@/assets/icons";

export function TickIcon({ width = 14, height = 11, color = "#FF7512" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 14 11" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path d="M2 4.86L5.42105 8.5L12 1.5" stroke={color} stroke-width="3" stroke-linecap="round" />
    </Svg>
  );
}
