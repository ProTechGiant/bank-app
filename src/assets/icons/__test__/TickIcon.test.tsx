import { create } from "react-test-renderer";

import { TickIcon } from "../TickIcon";

describe("icon/TickIcon", () => {
  it("renders a TickIcon", () => {
    const result = create(<TickIcon />);

    expect(result).toMatchSnapshot();
  });
});
