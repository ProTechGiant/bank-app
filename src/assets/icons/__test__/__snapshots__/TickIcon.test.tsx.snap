// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/TickIcon renders a TickIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="11"
  bbWidth="14"
  fill="none"
  focusable={false}
  height={11}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 11,
        "width": 14,
      },
    ]
  }
  vbHeight={11}
  vbWidth={14}
  width={14}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M2 4.86L5.42105 8.5L12 1.5"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4294931730,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
